src/ajv.js

```diff
@@ -60,7 +60,7 @@ function _ajvFactory(
 async function validate(data, schema, strictMode, cache, resolver) {
   const ajv = _ajvFactory(schema, strictMode, cache, resolver);
   addFormats(ajv);
-  const validateFn = await ajv.compileAsync(schema);
+  const validateFn = await ajv.compileAsync(schema, true);
```

Comment:
Can you explain the reason for adding this?

I'm not saying it is wrong - I'm just trying to gauge any knock-on effects of adding it.

It is added in a commit that adds tests, but if I remove it, all the tests pass so it seems like it might be an unrelated change.

Reply:
Geez, I think it's a leftover. I'll remove that.